HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -196,12 +202,23 @@ def customiseFor29658(process):
 
     return process
 
+# Use 8p fit (PR29617) and don't apply HB- correction (PR26177)
+def customiseFor29617(process):
+    for producer in producers_by_type(process, "HBHEPhase1Reconstructor"):
+        producer.algorithm.applyLegacyHBMCorrection = cms.bool( False )
+        from Configuration.Eras.Modifier_run2_HE_2017_cff import run2_HE_2017
+        from Configuration.Eras.Modifier_run3_HB_cff import run3_HB
+        #--- >= Run3 modification:
+        (run3_HB & run2_HE_2017).toModify(producer.algorithm, chiSqSwitch = -1)
+    return process
+
 # CMSSW version specific customizations
 def customizeHLTforCMSSW(process, menuType="GRun"):
 
     # add call to action function in proper order: newest last!
     # process = customiseFor12718(process)
     process = customiseFor29512(process)
     process = customiseFor29658(process)
+    process = customiseFor29617(process)
```

Comment:
it seems like the menuType can be passed to the customiseForNNNN, which would apparently be the better solution in the context of not using eras

Reply:
Remember, this customisation code is just temporary. Eventually, we'll move the customisations directly into the menu in ConfDB and re-generate the HLT menu dumps.
The development menus in CMSSW_11_1 now must correspond to run3.
Workflow 158.0 is using HLT:HIon, so either it is updated to/replaced by a Run3 workflow (it is MC after all), OR it should be modified to use the HLT fake2 menu.